nebula.config.js

```diff
@@ -20,6 +20,10 @@ module.exports = {
       DATA_BINNING: true,
       NUM_BUBBLES: true,
       PROGRESSIVE_RENDERING: true,
+      CLIENT_IM_3050: true,
+      SENSECLIENT_PS_17478: true,
+      STYLINGPANEL_BARCHART: true,
+      CLIENT_IM_747: true,
```

Comment:
I don't think we need all these flags for scatter-plot. Everyone except CLIENT_IM_3050 should be irrelevant.

Reply:
Done